### PR TITLE
fix(components): [el-switch] increase the word count of (in)active-text

### LINKED ISSUE #6112: [Bug Report] Switch 组件 active-text 和 inactive-text 只支持单个文字，两个或三个文字不支持
### Element Plus version
2.0.2

### OS/Browsers version
Chrome 98.0.4758.102

### Vue version
3.2.31

### Reproduction Link
https://codepen.io/sunsboyxu/pen/Vwrywya

### Steps to reproduce
根据 elemen-plus 文档，swtich 目前支持 active-text 和 inactive-text，switch组件 active-text 和 inactive-text props 多个以上的文字就会重现

### What is Expected?
希望 switch 组件 active-text 和 inactive-text 能够支持 两个文字或者3个文字
如：是否启动：启动 - 禁用

### What is actually happening?
# element-plus@2.0.1
### 查看的组件源码
``` js
// node_modules/element-plus/es/components/swtich/src/switch2.mjs
// createElementVNode("span", {}, [])

_ctx.activeText ? (openBlock(), createElementBlock("span", {
            key: 0,
            class: normalizeClass([_ctx.ns.is("text"), _ctx.checked ? _ctx.ns.is("show") : _ctx.ns.is("hide")]),
            "aria-hidden": !_ctx.checked
          }, toDisplayString(_ctx.activeText.substr(0, 1)), 11, _hoisted_4)) : createCommentVNode("v-if", true),
          _ctx.inactiveText ? (openBlock(), createElementBlock("span", {
            key: 1,
            class: normalizeClass([_ctx.ns.is("text"), !_ctx.checked ? _ctx.ns.is("show") : _ctx.ns.is("hide")]),
            "aria-hidden": _ctx.checked
          }, toDisplayString(_ctx.inactiveText.substr(0, 1)), 11, _hoisted_5)) : createCommentVNode("v-if", true)
        ], 64)) : createCommentVNode("v-if", true)
```
只取了，props 参数 active-text 和 inactive-text 的一位，是否支持2位或3位字符串
这里把 _ctx.activeText.substr(0, 1) 和 _ctx.inactiveText.substr(0, 1) 的 substr 方法去掉，样式需要调整下

```
.el-switch {
  .el-switch__inner {
     left: 6px;
     width: 100%;
    white-space: nowrap;
}
}
```

### PR DESCRIPTION
**Pull request by Alanscut**
fix: #6112 
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/switch/src/switch.vue [hunk 1]
```diff
@@ -57,14 +57,14 @@
             :class="[ns.is('text'), checked ? ns.is('show') : ns.is('hide')]"
             :aria-hidden="!checked"
           >
-            {{ activeText.substr(0, 1) }}
+            {{ activeText.substr(0, 3) }}
```
Comment (btea) [inline packages/components/switch/src/switch.vue:60]:
```suggestion
            {{ activeText.substring(0, 3) }}
```

Comment (Alanscut) [reply packages/components/switch/src/switch.vue:60]: 从0位开始截取字符串这两个方法功能上没有区别吧

Comment (btea) [reply packages/components/switch/src/switch.vue:60]: 是的，功能上没区别，不过`substr`被标记为`deprecated `，所以建议替换一下~

Comment (Alanscut) [reply packages/components/switch/src/switch.vue:60]: 哦，那我改一下

**Review by btea — APPROVED** [review]

**Review by jw-foss — APPROVED** [review]

**Pull request merged**